File: src/test/java/com/mpe85/grampa/rule/impl/TestNotRuleTest.java
package com.mpe85.grampa.rule.impl;

import static org.junit.jupiter.api.Assertions.assertTrue;

import org.junit.jupiter.api.Test;

public class TestNotRuleTest {
	
	@Test
	public void isPredicate() {
		assertTrue(new TestNotRule<>(new EmptyRule<>()).isPredicate());
	}
	
}

File: src/test/kotlin/com/mpe85/grampa/rule/impl/TestNotRuleTest.kt
package com.mpe85.grampa.rule.impl

import org.junit.jupiter.api.Assertions.assertTrue
import org.junit.jupiter.api.Test

class TestNotRuleTest {
  @Test
  fun isPredicate() {
    assertTrue(TestNotRule(EmptyRule<Unit>()).testRule)
  }
}